actions/codelist/analysis/generate_measures.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
```

Comment:
This module was refactored around a month ago. Would be good to use the latest version of the code, especially because it was changed to be more testable.

Reply:
I copied the code from interactive repo. Was there somewhere else?

Reply:
Hmm, it seems test-point-and-click and interactive diverge quite a bit

Reply:
Once your change is in and being used, then we should archive the test-point-and-click repo to save confusion